azure/table_azure_key_vault.go

```diff
@@ -146,6 +153,13 @@ func tableAzureKeyVault(_ context.Context) *plugin.Table {
 				Hydrate:     getKeyVault,
 				Transform:   transform.FromField("Properties.NetworkAcls"),
 			},
+			{
+				Name:        "private_endpoint_connections",
+				Description: "List of private endpoint connections associated with the key vault.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getKeyVault,
+				Transform:   transform.FromField("Properties.PrivateEndpointConnections"),
```

Comment:
PrivateEndpoint details are not coming, need to break this json. reference - azure_compute_disk_access PR.